examples/fluids/problems/blasius.c

```diff
@@ -76,11 +78,14 @@ PetscErrorCode modifyMesh(DM dm, PetscInt dim, PetscReal growth, PetscInt N,
 PetscErrorCode NS_BLASIUS(ProblemData *problem, DM dm, void *ctx) {
 
   PetscInt ierr;
-  User              user = *(User *)ctx;
-  MPI_Comm          comm = PETSC_COMM_WORLD;
-  BlasiusContext    blasius_ctx;
+  User           user     = *(User *)ctx;
+  MPI_Comm       comm     = PETSC_COMM_WORLD;
+  PetscBool      stg_bool = PETSC_FALSE;
```

Comment:
Our convention for booleans is `verb_noun`, so probably in this case `use_stg`?

Reply:
Yeah, that makes sense. Should I also change `mean_only` to `use_mean_only` or `use_only_mean`?

Also, maybe change the flag in `blasius.c` to `-stg_use` (so that all the stg-related flags can use the:

```yaml
stg:
  use: true
  stg_inflow_path: './path/STGInflow.dat'
  ...[etc]
```

Reply:
Yeah, `use_mean_only` would be consistent

for the command line flag I'm fine with the current

Reply:
Should STG be a sub-type of Blasius or its own problem type (that might call Blasius setup and then do extra stuff)? If it's a subtype, then maybe `-blasius_inflow_type stg`. Or maybe the problem type should become `plate` with `-plate_inflow_type [blasius | stg]`?